recipes/pyaaf2/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pyaaf2" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 160d3c26c7cfef7176d0bdb0e55772156570435982c3abfa415e89639f76e71b
+
+build:
+  number: 0
```

Comment:
It looks like this package could be "noarch": https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

just add `noarch: python`